docker/dev/Dockerfile.npm

```diff
@@ -0,0 +1,17 @@
+# We use the official Python images. This may need to get locked in, or back to Ubuntu at some point
+FROM node
+
+RUN mkdir /code
+
+WORKDIR /code
+COPY . .
+
+RUN npm install
+RUN node_modules/.bin/bower --allow-root install
+RUN node_modules/.bin/grunt build
```

Comment:
`grunt dev` when doing development - this ensures the scripts are not minified, helping debug problems.

Also, `npm run dev` is missing. This builds the webpack bundle. Failing to do so will crash the UI when trying to access a Vue view.